coordinator/src/services/translation/request/agent-unattended.ts

```diff
@@ -3,58 +3,72 @@ import {getIdentifierValueForSystem} from "../common"
 import {convertAddress, convertTelecom} from "./demographics"
 import pino from "pino"
 import {odsClient} from "../../communication/ods-client"
+import Hapi from "@hapi/hapi"
+import {
+  getRoleCode,
+  getSdsRoleProfileId,
+  getSdsUserUniqueId,
+  getUserName
+} from "../../../utils/headers"
 
 export async function createAuthorForUnattendedAccess(
```

Comment:
No longer anything to do with unattended access. Maybe `createAuthorFromAuthenticatedUserDetails` or something? Names are hard